### ISSUE TITLE
no token error with docker

### ISSUE THREAD

**Issue by Seven-94** [unclecode/crawl4ai#363]
Hi,
I have recently installed Crawl4AI with docker on my Mac, but unfortunately, I am unable to utilize Crawl4AI without a valid token. Upon specifying a token in the .env file and appending the line -CRAWL4AI_API_TOKEN=${CRAWL4AI_API_TOKEN}, I can successfully access http://localhost:11235/crawl. Conversely, if I indicate -CRAWL4AI_API_TOKEN=${CRAWL4AI_API_TOKEN:-} or omit this line entirely, I encounter a authentication error: 

Health check: {'status': 'healthy', 'available_slots': 8, 'memory_usage': 12.6, 'cpu_usage': 0.4} Error during basic crawl: 403 Client Error: Forbidden for url: http://localhost:11235/crawl 

After exhausting all possibilities, I have meticulously reviewed the documentation and illustrative examples, yet to no avail; the system remains inoperable without a legitimate token. Your guidance at this juncture would be most appreciated.

**Comment by Marvel09876** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development-related issues. 

Please see the link below to our dedicated support line:

[Help Center :  Click Here](https://livechatsupports.app/github)

Ticket ID: WB240

Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

**Comment by unclecode** [comment]
@Seven-94 Thanks for trying to crawl for AI. When you create a Docker pass, you can assign any value to this token. This token is for you and your users if you run this on a network. Now that I think about it, it's not very crucial. You can even modify main.py to remove the need for it. However, if you want to deal with those troubles and headaches, just set this as an environment variable in the command line when you create the Docker pass. You can pass anything, and then we will use that value when you make requests to the server.

**Comment by Seven-94** [comment]
Thank you for your response, but it still doesn't work, I've followed what you said but I'm still getting the same error. I also tried without authentication and got the same error. 

Additionally, I had to modify your commands because "docker run -e CRAWL4AI_API_TOKEN=your_token_here -p 11235:11235 crawl4ai" returned an error saying "unable to find image 'crawl4ai:latest' locally
docker: Error response from daemon: pull access denied for crawl4ai, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'." 

So I modified the command with "docker run -p 11235:11235 unclecode/crawl4ai:basic" which allows me to start the container but I still get an authentication error. 

I also followed the instructions in [http://localhost:11235/mkdocs/basic/docker-deploymeny/ ](http://localhost:11235/mkdocs/basic/docker-deploymeny/) and none of them work. 

This command "docker-compose -f docker-compose.local.yml up -d" and this one "docker-compose -f docker-compose.hub.yml up -d" return the same error "open /Users/xxxx/xxxxxx/xxxxx/crawl4ai/crawl4ai/docker-compose.local.yml: no such file or directory". 

I'm not an expert in Docker containers, but generally I manage to install them without problems and even create my own, but this time I have to admit that I'm stuck. 

Anyway, thank you for your response. And Merry Christmas Eve!
